Source/Krypton Components/Krypton.Ribbon/Palette/ContextTabSet.cs

```diff
@@ -21,7 +21,7 @@ internal class ContextTabSet
     {
         #region Instance Fields
 
-        private ViewDrawRibbonTab _lastTab;
+        private ViewDrawRibbonTab? _lastTab;
```

Comment:
I do not like these sort of changes.. Because if the class is just ignoring the nullable by the use of `!`. then why have it has a nullable object ?

Please back these out.

Reply:
Thanks for reviewing, code corrected as requested including a couple of additional modifications I came across.
Warnings slightly reduced. 

See https://github.com/Krypton-Suite/Standard-Toolkit/pull/1443/commits/a63cae17d8b1c6a031da7d6c97b54002b87bb8f9

![compile-results](https://github.com/Krypton-Suite/Standard-Toolkit/assets/96108132/d7b6b828-19b3-4212-9e59-4d2139f24b83)